helloworld/.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2
```

Comment:
I dont think you need the `elloworld/.circleci/config.yml` file anymore

Reply:
@tovbinm  i know that, will remove it